resources/contenttweaker/lang/en_us.lang

```diff
@@ -253,27 +255,88 @@ e2ee.tconstruct.material.researcher.name=Researcher
 e2ee.tconstruct.material.researcher.description=Based on your research in thaumonomicon, there are various effects!
 e2ee.tconstruct.material.first_stand.name=First Stand
 e2ee.tconstruct.material.first_stand.description=Consume experience before taking damage.
+e2ee.tconstruct.material.robust.name=Robust
```

Comment:
And additions to any .lang files should be reflected in all other .lang files for other languages for easier syncing and translating.
Also, im highly recommend to use ChatGPT 4o, since its output very good result with low effort. Just tell it "Im making modpack, there is my goal, there is context. Now, translate this block of lang codes into ru_ru and zh_cn"